scripts/binary/trigger-publish-binary-pipeline.js

```diff
@@ -10,6 +10,7 @@ const { getNextVersionForBinary } = require('../get-next-version')
   const { nextVersion } = await getNextVersionForBinary()
 
   const body = JSON.stringify({
+    branch: 'cacie/upgrade-electron-27',
```

Comment:
do we get rid of this retroactively after we merge or does this need to be done before we merge into develop?

Reply:
I think we'll need to merge the publish binary first and then this can go away. This is an interesting point though. Once we merge the publish binary, it'll be used for all PRs and will break for older PRs. Curious if we've gone through this kind of non passive change before? @astone123?

Reply:
@ryanthemanuel and I discussed this yesterday - I think the process should be
- merge this Cypress PR **with** this branch name specified
- once this change is more propagated into development branches, merge [publish binary PR](https://github.com/cypress-io/cypress-publish-binary/pull/56)
- remove the branch name in this script